### PR TITLE
implemented copy method config to agora

### PR DESCRIPTION
**Pull request by dvoet**
https://broadinstitute.atlassian.net/browse/DSDEEPB-422

- [x] **Submitter**: Rebase to master. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Make sure documentation for code is complete
- [x] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [x] **Submitter**: **Assign PR to** 
- [x] : does a once-over of the PR
- [x] : Anoint a lead reviewer (LR). **Assign PR to LR**
- [x] **LR**: Initial review by LR and others.
- [ ] Comment / review / update cycle:
  * Rest of team may comments on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter updates documentation as needed
  * Submitter rebases to master again if necessary
  * Submitter makes further commits. DO NOT SQUASH. **Reassign to LR** for further feedback
- [ ] **LR**: sign off, **assign to submitter** to finalize
- [ ] **Submitter**: Squash commits, rebase if necessary
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Merge to master
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: Check configuration files in Jenkins in case they need changes
- [ ] **Submitter**: Verify swagger UI on dev server still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/main/resources/swagger/rawls.yaml [hunk 1]
```diff
@@ -1738,6 +1761,23 @@ definitions:
         $ref: '#/definitions/MethodConfigurationName'
         description: The destination for a copied method configuration
 
+  MethodRepoConfigurationExport:
+    description: ''
```
Comment (abaumann) [inline src/main/resources/swagger/rawls.yaml]: add description

File: src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpMethodRepoDAO.scala [hunk 2]
```diff
@@ -37,9 +47,19 @@ class HttpMethodRepoDAO( methodRepoServiceURL: String )( implicit system: ActorS
 
   private def when500( throwable: Throwable ): Boolean = {
     throwable match {
-      case ure: spray.client.UnsuccessfulResponseException => ure.responseStatus.intValue/100 == 5
-      case ure: spray.httpx.UnsuccessfulResponseException => ure.response.status.intValue/100 == 5
+      case ure: spray.client.UnsuccessfulResponseException => ure.responseStatus.intValue / 100 == 5
```
Comment (abaumann) [inline src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpMethodRepoDAO.scala]: one case statement with different types, same action?

**Comment by abaumann** [comment]
looks good, back to you if you are going to update after my comments

File: src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpMethodRepoDAO.scala (hunk 2 above)
Comment (tedsharpe) [inline src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpMethodRepoDAO.scala]: I think it's a mistake to remove the httpx case.  Which you'll receive depends on whether the exception is thrown by sendReceive or unmarshal.

File: src/main/resources/swagger/rawls.yaml (hunk 1 above)
Comment (dvoet) [inline src/main/resources/swagger/rawls.yaml]: none of the others have a description, I was only trying to maintain consistency

**Pull request merged**